corehq/apps/data_cleaning/tests/test_filters.py

```diff
@@ -941,3 +941,57 @@ def test_default_pinned_filters(self):
         assert case_owners_filter.value is None
         case_status_filter = self.session.pinned_filters.get(filter_type=PinnedFilterType.CASE_STATUS)
         assert case_status_filter.value is None
+
+
+class TestBulkEditFilterManagers(TestCase):
+    domain = 'test-bulk-edit-filter-managers'
+
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        cls.domain_obj = create_domain(cls.domain)
+        cls.addClassCleanup(cls.domain_obj.delete)
+
+        cls.web_user = WebUser.create(cls.domain, 'tester@datacleaning.org', 'testpwd', None, None)
+        cls.addClassCleanup(cls.web_user.delete, cls.domain, deleted_by=None)
+
+    def setUp(self):
+        super().setUp()
+        self.session = BulkEditSession.objects.new_case_session(
+            self.web_user.get_django_user(),
+            self.domain,
+            'plants',
+        )
+
+    def test_copy_filters_to_session(self):
+        self.session.add_filter(
+            'soil_contents',
+            DataType.MULTIPLE_OPTION,
+            FilterMatchType.IS_ANY,
+            'bark worm_castings',
+        )
+        new_session = BulkEditSession.objects.new_case_session(
+            self.web_user.get_django_user(),
+            self.domain,
+            'plants',
+        )
+        self.session.filters.copy_to_session(self.session, new_session)
+        assert new_session.filters.count() == self.session.filters.count()
+        other_session_filter = new_session.filters.first()
```

Comment:
nit: `other_session_filter` is used here but `new_pinned_filter` is used below - I would prefer to see either `other` or `new` used for both unless there's a distinction I'm not catching on to.